spec/functional/specific_pages_titles_spec.rb

```diff
@@ -1,9 +1,11 @@
 # frozen_string_literal: true
 
 require 'spec_helper'
+require_relative '../../lib/testing-helpcenter/data/page_title_list'
```

Comment:
It's better to require this file in the same place as all other similar files in this folder, try to find this place

Requiring something only in `spec` file is not a good practice